scss/_settings_spacing.scss

```diff
@@ -53,6 +53,7 @@ $spv-nudge-compensation: $sp-unit - $spv-nudge !default;
 $spv-intra--condensed: $sp-unit * .5 !default; // buttons margins inside table,
 $spv-intra--condensed--scaleable: $spv-intra--condensed * $multi !default;
 $spv-intra: $sp-unit !default;
+$spv-intra--regular: $sp-unit * 2; //card padding
```

Comment:
Please remove this comment. We may use this variable for other patterns in the future.